### PR TITLE
[BIOMAGE-2350] - Add `public_api_url` in dev to send in SNS messages

### PR DESCRIPTION
**Pull request by aerlaut**
# Description
`public_api_url` is required in the SNS message body by the endpoint lambda to route messages to the correct endpoint. We already have this for staging and production. This PR configures the value in development.

Staging: https://ui-agi-pipeline121.scp-staging.biomage.net/

# Details
#### URL to issue
https://biomage.atlassian.net/browse/BIOMAGE-2350

#### Link to staging deployment URL (or set N/A)
N/A

#### Links to any PRs or resources related to this PR
N/A

#### Integration test branch
master

# Merge checklist
Your changes will be ready for merging after all of the steps below have been completed.

### Code updates
Have best practices and ongoing refactors being observed in this PR
- [ ] Migrated any selector / reducer used to the new format.

### Manual/unit testing
- [ ] Tested changes using InfraMock locally **or** no tests required for change, e.g. Kubernetes chart updates.
- [ ] Validated that current unit tests for code work as expected and are sufficient for code coverage **or** no unit tests required for change, e.g. documentation update.
- [ ] Unit tests written **or** no unit tests required for change, e.g. documentation update.

### Integration testing
**You must check the box below** to run integration tests on the latest commit on your PR branch.
Integration tests have to pass before the PR can be merged. Without checking the box, your PR
**will not pass** the required status checks for merging.

- [ ] Started end-to-end tests on the latest commit.

### Documentation updates
- [ ] Relevant Github READMEs updated **or** no GitHub README updates required.
- [ ] Relevant Wiki pages created/updated **or** no Wiki updates required.

### Optional
- [ ] Staging environment is unstaged before merging.
- [ ] Photo of a cute animal attached to this PR.

### CODE REVIEW

File: pipeline-runner/R/init-functions.R [hunk 1]
```diff
@@ -108,6 +108,7 @@ load_config <- function(development_aws_server) {
 
   if (config$cluster_env == "development") {
     config$api_url <- sprintf("http://%s:3000", development_aws_server)
+    config$public_api_url <- config$api_url
```
Comment (cosa65) [inline pipeline-runner/R/init-functions.R:111]: Do we even need `api_url` at this point? Now everything will use the public url, right?

Comment (aerlaut) [reply pipeline-runner/R/init-functions.R:111]: Yea, they seem to be the same thing. I'll test if just using `api_url` works

Comment (aerlaut) [reply pipeline-runner/R/init-functions.R]: I tested in loal, staging, for both pipeline pod and batch, and replacing it with just `api_url` works

**Review by cosa65 — APPROVED** [review]

File: pipeline-runner/R/init-functions.R (hunk 1 above)
Comment (cosa65) [reply pipeline-runner/R/init-functions.R]: nice!

**Pull request merged**